src/style.css

```diff
@@ -1 +1,112 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url("/src/fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-heavy;
+  src: url("/src/fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-weight: 300;
+}
+
+.form-search {
+  display: flex;
+  align-items: center;
+  justify-content: start;
+}
+
+.big {
+  margin-top: 20px;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
```

Comment:
it is better to choose one color format and use it everywhere in your project. choose either hex or RGB and use it